src/Pokemon.java

```diff
@@ -0,0 +1,69 @@
+import java.util.ArrayList;
+
+public abstract class Pokemon {
+
+    private String name;
+    private int level;
+    private int hp;
+    private String food;
+    private String sound;
+    protected String type;
+    protected ArrayList<String> attacks = null;
```

Comment:
type en attacks kun je ook final maken, maar dan moet je ze nog wel aan je constructor toevoegen, in plaats van dat `this.type = ...` doet in je subklassen